db/migrate/20190527191030_create_device_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateDeviceTable < ActiveRecord::Migration[5.2]
```

Comment:
what this file is doing here?

Reply:
this PR is only about adding DeviceGroups

Reply:
as i understand in the past someone deleted the devices from schema, so i recreated them to pass all rspec tests